lessons/introR/lesson.md

```diff
@@ -109,10 +110,10 @@ datasets that you can practice on.  The one I'm going to use is the
 `swiss` dataset.
 
 
-~~~r
```

Comment:
OK, maybe this is where I should ask: Is this necessary? It appears that the existing pages is formatted correctly, no?